### PR TITLE
Made site better

### PR DESCRIPTION
**Pull request by StarNumber12046**
The site looks better

### CODE REVIEW

**Comment by user0-07161** [comment]
Amazing

**Review by user0-07161 — CHANGES_REQUESTED** [review]
Would you like to replace the download link with `https://github.com/Windivs-Devs/windivs/releases`, please?

**Comment by user0-07161** [comment]
In line 15, index.html.

File: index.html [hunk 1]
```diff
@@ -3,74 +3,44 @@
...
+      
+      <header>
         <nav>
-            <ul>
-                <li><a href="#home">Home</a></li>
-                <li><a href="#features">Features</a></li>
-                <li><a href="https://github.com/Windivs-Devs/windivs/releases/tag/6.01.6568.1.7dev" class="cta-button">Try WinDivs Today</a></li>
-            </ul>
+          <a href="#gallery">Gallery</a>
+          <a href="https://github.com/Windivs-Devs/windivs/releases/tag/6.01.6568.1.7dev" id="download">Download</a>
```
Comment (user0-07161) [inline index.html:15]: Replace the page with https://github.com/Windivs-Devs/windivs/releases

**Review by user0-07161 — CHANGES_REQUESTED** [review]
Well, this is the correct way to use the review thingy, I'm using it for the first time, so just ignore the first one.

**Comment by StarNumber12046** [comment]
done :)

**Comment by user0-07161** [comment]
Thanks!

**Review by user0-07161 — APPROVED** [review]

**Comment by errortek** [comment]
Great improvement! The site looks amazing!! Well done @StarNumber12046 !

**Pull request merged**